openbb_platform/providers/tmx/openbb_tmx/models/index_sectors.py

```diff
@@ -0,0 +1,90 @@
+"""TMX Index Sectors fetcher."""
+
+# pylint: disable=unused-argument
```

Comment:
@IgorWounds is there any reason we cant just add this to pylint.rc ?

Reply:
Well, it would then stop complaining where it matters. Maybe we could place it under all providers/* ?